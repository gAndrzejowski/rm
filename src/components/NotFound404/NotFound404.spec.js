import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import NotFound404 from './NotFound404';

describe('404 page component', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<NotFound404 />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
