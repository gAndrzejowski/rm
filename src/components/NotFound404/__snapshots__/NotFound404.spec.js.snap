// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page component renders correctly 1`] = `
<article
  className="notfound"
>
  <h2>
    We couldn't find this page...
  </h2>
  <img
    alt=""
    src="img/404.jpg"
  />
  <p>
    Try another url or use the search box.
  </p>
</article>
`;
